guides/v2.2/ui_comp_guide/components/ui-dynamicrows-record.md

```diff
@@ -0,0 +1,94 @@
+---
+group: ui-components-guide
+title: DynamicRowsRecord component
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The DynamicRowsRecord component is a container of record fields. The DynamicRowsRecord should be used as a child of [DynamicRows]({{ page.baseurl }}/ui_comp_guide/components/ui-dynamicrows.html) component.
+
+## Configuration options
+
+| Option | Description | Type | Default |
+| --- | --- | --- | --- |
+| `disabled` | Initial component's state. When set to `true`, users can't take action on the element. | Boolean | `true` |
+| `headerLabel` | The label of the record. It is used as a record `label` in case if the `label` option is not initialized. | String | `''` |
```

Comment:
Does that mean that `headerLabel` value is just a fallback for `label` value? Are you sure that's the only purpose of this field? 
Maybe it is, I'm just asking because having this flow looks a little bit weird.

Reply:
Thank you for the clarification